Backend/dev/migrations/dynamic-offer-driver-app/0247-cosmetic-fixes-in-go-home-columns.sql

```diff
@@ -1,3 +1,3 @@
-ALTER TABLE atlas_driver_offer_bpp.go_home_config RENAME COLUMN go_home_req_id TO go_home_request_id;
```

Comment:
@Vignesh-772 Why is migration altered

Reply:
Yes, @kavya-shree-s we can keep it, actually this migration was wrongly pushed to main so locally main was throwing error.